Doc/library/socket.rst

```diff
@@ -69,6 +69,13 @@ created.  Socket addresses are represented as follows:
   where *host* is a string representing either a hostname in Internet domain
   notation like ``'daring.cwi.nl'`` or an IPv4 address like ``'100.50.200.5'``,
   and *port* is an integer.
+  
+  - For IPv4 addresses, two special forms are accepted instead of a host 
```

Comment:
You've got trailing whitespace on some of these lines, aside from that the content looks good.